kernel/packages/config/index.ts

```diff
@@ -252,7 +256,7 @@ export function getServerConfigurations() {
   return {
     contentAsBundle: `https://content-assets-as-bundle.decentraland.org`,
     wearablesApi: `https://wearable-api.decentraland.org/v2`,
-    explorerConfiguration: `https://explorer-config.decentraland.${
+    explorerConfiguration: `http://explorer-config.decentraland.${
```

Comment:
```suggestion
    explorerConfiguration: `https://explorer-config.decentraland.${
```
Fails in production otherwise!

Reply:
Gone back to HTTPS!